app/dts/behaviors/homerow_tap.dtsi

```diff
@@ -0,0 +1,12 @@
+/ {
+	behaviors {
+		ht: behavior_homerow_mod {
+			compatible = "zmk,behavior-hold-tap";
+			label = "homerow_mod";
```

Comment:
Nitpick: The convention in Zephyr seems to be uppercase for the labels used for registering driver instances, so perhaps:

```suggestion
			label = "HOMEROW_MOD";
```